### DISCUSSION TITLE
Contribution threshold for the certificate

### DISCUSSION THREAD

**Discussion by Yodha-Sudarsi** [sanjay-kv/Open-source-Practice#1011] [Q&A] [answered]
What is the minimum contribution requirement for obtaining a certificate?

**Comment by Debayan-Ghosh2005** [comment]
if you on the leaderboard you will get an certificate.

**Comment by sanjay-kv** [comment, marked as answer]
@Yodha-Sudarsi https://github.com/GSSoC24/Contributor/discussions/33
